src/rules/no-complex-cloud-params.ts

```
@@ -132,8 +132,22 @@ export const noComplexCloudParams = createRule<[], MessageIds>({
             return isClassInstance(def.node.init);
           }
         }
-        // Check if the identifier starts with a capital letter (potential class instance)
-        return node.name[0] === node.name[0].toUpperCase();
+
+        // Only consider identifiers as potential class instances if they follow PascalCase naming
+        // (not SCREAMING_SNAKE_CASE constants)
+        if (node.name[0] === node.name[0].toUpperCase()) {
+          // Check if it's a constant (SCREAMING_SNAKE_CASE) - these are not class instances
+          if (node.name.includes('_') && node.name === node.name.toUpperCase()) {
```

ESLint

[error] Replace `node.name.includes('_')·&&·node.name·===·node.name.toUpperCase()` with `⏎············node.name.includes('_')·&&⏎············node.name·===·node.name.toUpperCase()⏎··········`

(prettier/prettier)